src/file.ts

```diff
@@ -423,6 +423,7 @@ export interface FileMetadata extends BaseMetadata {
   };
   customTime?: string;
   eventBasedHold?: boolean | null;
+  readonly eventBasedHoldReleaseTime?: string;
```

Comment:
Where do we use/set this?

Reply:
It is not user setable. It is a field that will be added as a fast follow that is only returned if an event-based hold was released.